nsqadmin/static/js/views/app.js

```diff
@@ -45,11 +46,11 @@ var AppView = BaseView.extend({
         this.listenTo(Pubsub, 'view:ready', function() {
             $('.rate').each(function() {
                 $.get(AppState.url('/graphite'), {
-                    'metric': 'rate',
-                    'target': $(this).attr('target')
-                }).done(function(data) {
-                    $(this).html(data['rate']);
-                }.bind(this));
+                        'metric': 'rate',
+                        'target': $(this).attr('target')
+                    })
+                    .done(function(data) { $(this).html(data['rate']); }.bind(this))
+                    .fail(function() { $(this).html('ERROR'); }.bind(this));
             });
```

Comment:
If instead of relying on jQuery's special handling of `this` (which can get confusing - especially when nested within several functions), you can be a little more explicit (and lose the `bind`s in the process) if you do something like this:

```
$('.rate').each(function(i, el) {
    $.get(AppState.url('/graphite'), {
        'metric': 'rate',
        'target': $(el).attr('target')
    })
    .done(function(data) { $(el).html(data['rate']); })
    .fail(function() { $(el).html('ERROR'); });
});
 ```

or even:

```
$('.rate').each(function(i, el) {
    var $el = $(el);
    $.get(AppState.url('/graphite'), {
        'metric': 'rate',
        'target': $el.attr('target')
    })
    .done(function(data) { $el.html(data['rate']); })
    .fail($el.html.bind($el, 'ERROR'));
});
 ```